### ISSUE TITLE
Forgot header file

### ISSUE THREAD

**Issue by troy128k** [NerdsvilleCEO/go-c-sum#1]
Hello, just thought I'd point out that you forgot the header to your C code? Not sure if that was intentional.

**Comment by NerdsvilleCEO** [comment]
Agreed! I could add this, but this is just a really simple sample of using go-c. This will be migrated to a library of personal experimental stuff.
